shared/src/lib.rs

```diff
@@ -124,29 +134,100 @@ pub type EADMessageBuffer = EdhocMessageBuffer; // TODO: make it of size MAX_EAD
 ///
 /// Semantically, this is a byte string of some length.
 ///
-/// This currently only supports numeric values (see
-/// https://github.com/openwsn-berkeley/lakers/issues/258), but may support larger values in the
-/// future.
+/// Its legal values are constrained to only contain a single CBOR item that is either a byte
+/// string or a number in -24..=23, all in preferred encoding.
```

Comment:
preferred encoding?

Reply:
CBOR could say 23 as h'17' as well as '1817' or '190017' or '1a00000017'; in CBOR we have to use preferred encoding / preferred serialization / deterministic encoding (terminology is subject to current drafts).